about.html

```diff
@@ -0,0 +1,123 @@
+                    <li class="nav-item"><a href="#" class="nav-link">Sponser</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                </ul>
+    
+
+        <section class="about-headline d-flex flex-column gap-3 justify-content-between  align-items-center text-center  p-5 m-auto">
+            <div><h3 class=" h-grd">CREATIVE COMNMONS<br>GLOBAL EVENT 2023</h3></div>
+            <p class="about-para p-font border border-3 border-light p-3 bg-white">This Event will show us that the next big thing isn’t a new technology, it’s a new idea.
+                 Human Security and technology<br> in all lief aspect is that idea. The Human Security concept also includes access to healthcare, <br>economic security, environmental 
+                 protection including resiliency and sustainability, personal safety and mobility, community security,<br> access to education,
+                  broadband access and political freedom</p>
+                <span class=" text-muted"> Please contact us per Email for nay further questions about CC Global Summit 2015! </span>
+                  <p class="mt-2  text-muted" > techno2023@cckorea.org</p>
+        </section>
+        </div>
+
+        <section class="container-fluid p-5 bg-light logo">
+            <div class="d-flex flex-column flex-wrap gap-5 align-items-center">
+                <h3 class="text-center">Creative Commons Global Event 2023 Logo</h3>
+                <p class="text-center">The logo of TechNo Global Event 2023 was decided through the logo competition from 8, June to 7, July.<br>
+                    After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.<br>
+                    After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s event as follows:</p>
+                    <img src="imgs/logo.jpg" width="300px" alt="logo img"/>
+            </div>
+        </section>
+         
+        <section class=" container-fluid bg-white">
+            <div class="d-flex flex-column gap-2 p-5 m-auto justify-content-around align-items-center">
+            <h3 class="text-center">See the past CC Global Summits</h3>
+            <p class="text-center">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            <div class="d-sm-flex flex-lg-row flex-md-row flex-sm-column align-items-center justify-content-between gap-3 about-past">
+                <img src="imgs/paris.png" alt="first event " width="432px" height="236px">
+                <img src="imgs/morocco.png" alt="second event" width="432px" height="236px">
+            </div>
+            </div>
+        </section>
+      
+          <!-----footer section-->
+          <section class="partners container-fluid ">
+            <h3 class="text-center text-dark p-lg-5 p-md-4 p-sm-2 fw-bolder">Partners</h3>
```

Comment:
- [ ] PLease make the color of partner white since in your design is black and in the original design is white so please change it 🤩✨✨ and please decrease the font size since in your design it is bigger but it is smaller in the given✨✅

- Yours
![image](https://user-images.githubusercontent.com/58553711/227086247-98f1a3d1-8c9f-495e-ab77-108453a664e9.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/227086294-cf736368-c2e0-478f-911b-21816ffa4cb7.png)